### ISSUE TITLE
Will Pajarito be updated to work with JuMP 0.20?

### ISSUE THREAD

**Issue by ierhard** [JuliaOpt/Pajarito.jl#430]
Will Pajarito be updated to work with JuMP 0.20?

This is what I see when I try to install Pajarito via `Pkg.add("Pajarito")`:
```
Unsatisfiable requirements detected for package JuMP [4076af6c]:
JuMP [4076af6c] log:
 ├─possible versions are: [0.18.3-0.18.6, 0.19.0-0.19.2, 0.20.0-0.20.1] or uninstalled
 ├─restricted to versions 0.20.1 by an explicit requirement, leaving only versions 0.20.1
 └─restricted by compatibility requirements with Pajarito [2f354839] to versions: 0.18.3-0.18.6 — no versions left
   └─Pajarito [2f354839] log:
     ├─possible versions are: 0.6.1 or uninstalled
     └─restricted to versions * by an explicit requirement, leaving only versions 0.6.1
```
If there's another way to use Pajarito at the moment, I'd be grateful to hear your advice.

**Comment by ccoffrin** [comment]
This update has been blocked by adding solver-agnostic callbacks which have been added to JuMP-master recently.  However, I am not aware of anyone who is actively working on porting it, community contributions would likely be welcome.

**Comment by jay-dave** [comment]
I have the same question. Has there been any update since the last post?

**Comment by odow** [comment]
No change. Help appreciated.

**Comment by odow** [comment]
Closing in favor of #411
